applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_ausas_solver.py

```diff
@@ -0,0 +1,111 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Check that applications were imported in the main script
+KratosMultiphysics.CheckRegisteredApplications("FluidDynamicsApplication")
+
+# Import applications
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+
+# Import base class file
+import navier_stokes_embedded_solver
+
+def CreateSolver(main_model_part, custom_settings):
+    return NavierStokesEmbeddedAusasMonolithicSolver(main_model_part, custom_settings)
+
+class NavierStokesEmbeddedAusasMonolithicSolver(navier_stokes_embedded_solver.NavierStokesEmbeddedMonolithicSolver):
+
+    def __init__(self, main_model_part, custom_settings):
+
+        #TODO: shall obtain the compute_model_part from the MODEL once the object is implemented
+        self.main_model_part = main_model_part
+
+        ##settings string in json format
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "solver_type": "EmbeddedAusas",
```

Comment:
I would remove this setting in the python wrapper (with `RemoveValue`), since it is not needed in the actual solver

Reply:
This would make the implementation more cumbersome and I don't see the problem in keeping that field in there. Besides, if it is removed, it dissappears when you print the ProjectParameters.json (something that might be useful when debugging).

Reply:
just to say that i am less and less convinced that the solver_wrapper is a good solution. I am NOT proposing to change it right now, but i less and less see the utlity of it.